services/app/apps/codebattle/assets/js/widgets/slices/game.js

```diff
@@ -33,6 +33,12 @@ const game = createSlice({
     setGameTask: (state, { payload: { task } }) => {
       state.task = task;
     },
+    deleteAlert: (state, { payload }) => {
+      delete state.alerts[payload];
+    },
+    addAlert: (state, { payload }) => {
+      Object.assign(state.alerts, payload);
```

Comment:
[nit](https://onetwobytes.com/2021/11/09/nit-code-review/): Лучше пользоваться свежим синтаксисом со спред оператором
```suggestion
      state.alerts = {...state.alerts, ...payload};
```